packages/elements/src/tree/managers/tree-manager.ts

```diff
@@ -430,23 +433,23 @@ export class TreeManager<T extends TreeDataItem> {
   }
 
   /**
-   * Expands all items in the tree
+   * Expands all items to show all child items.
    * @returns {void}
    */
   public expandAllItems(): void {
     this.parentItems.forEach((item) => this.expandItem(item));
   }
 
   /**
-   * Collapses all items in the tree
+   * Collapses all items to hide all  child items.
```

Comment:
The original comment is better as show items is not just abot `expand`, it's about `hidden` as well.